examples/admin_policy/example_server/policy_server.py

```diff
@@ -20,39 +21,24 @@ async def apply_policy(request: Request) -> JSONResponse:
     # Decode from request body
     json_data = await request.json()
     user_request = sky.UserRequest.decode(json_data)
+    # Change the following list to apply one or more policies to the user
+    # request. You can also implement custom policies in server in addition to
+    # loading policies from external packages.
+    policies: List[sky.AdminPolicy] = [
+        example_policy.SetMaxAutostopIdleMinutesPolicy,
+        example_policy.UseSpotForGpuPolicy,
+    ]
+    try:
+        for policy in policies:
+            mutated_request = policy.validate_and_mutate(user_request)
+            user_request.task = mutated_request.task
+            user_request.skypilot_config = mutated_request.skypilot_config
```

Comment:
No need to be this PR (can be for the future): instead of applying the chain of policy directly in the server side, I think the easier way for a user is being able to specify what policies to apply in the SkyPilot config, and SkyPilot will apply all of them automatically. For example,

```
admin_policy:
  host: policy-host-server-url
  policies:
  - set_autostop
  - use_spot_for_gpu
```

Reply:
Sounds good! How about use URL params instead of structured config?

```
admin_policy:
  host: policy-host-server-url
  params:
    policies: set_autostop,use_spot_for_gpu
    # It can also be included in the POST body if the server handles body
    # body: ...
```

Or simpler:

```
admin_policy: http://policy-host-server-url?policies=set_autostop,use_spot_for_gpu
```

This way, we can decouple the config with certain server impl, anyway, the server we provide is just an example and  users are free to decide how to parse the URL parameters and request body